### PR TITLE
Feature/qr code reader

### PR DESCRIPTION
**Pull request by AlexTeusz**
Azure Board PR Review: 
TASK 7293

### CODE REVIEW

**Comment by kwinto** [comment]
Sadly I could not get it to work given the error in console `Error in QR code scanner: NotFoundError: The object can not be found here.` Tested in desktop Firefox, mobile Safari and mobile Chrome with a standard data payload:
`{
  "_plugin": {
    "type": "qr-code"
  }
}`

If any additional data is necessary for plugin to work please provide this in comments.

**Comment by AlexTeusz** [comment]
There is an issue in the used QR code reader npm package: 
https://github.com/JodusNodus/react-qr-reader/issues/137

I have to wait until its fixed.

**Comment by mastasky** [comment]
@AlexTeusz - are there news on this? It seemed fine at the Telekom event

**Comment by AlexTeusz** [comment]
Yes, it worked. @kwinto can you please test it again and approve it ?

**Comment by AlexTeusz** [comment]
@kwinto  and @mastasky ,

I added documentation and tested the webchat plugin several times. It works :) 

The camera has to be visible, which means that the webcam of your computer has to be open. When you want to test this webchat plugin on your mobile device, the webchat has to be deployed via HTTPS.

Please test and approve. 
Thanks 

// Alex

File: plugins/qr-code-reader/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# QR Code Reader
+
+With this webchat plugin, you can read QR codes and send the scanned information back to the chatbot.
+In order to do so, you have to send a specifiy **SAY NODE** in Cognigy.AI which starts the plugin in your webchat:
```
Comment (kwinto) [inline plugins/qr-code-reader/README.md]: Did you mean **CODE NODE**?

Comment (AlexTeusz) [reply plugins/qr-code-reader/README.md]: you can do it with a say node or a code node

**Comment by kwinto** [comment]
It works, great job! One small thing - can you please fix the variables of the button labels? I think you mistakenly use SubmitButtonText on all the three buttons and currently they all get a text "Submit" instead of "Submit", "Cancel", "Open".

**Comment by AlexTeusz** [comment]
@kwinto done

**Comment by kwinto** [comment]
Well done!

**Pull request merged**